### ISSUE TITLE
Cannot select "nvprod" folder after Android 14 security update

### ISSUE THREAD

**Issue by rene-j** [Razviar/marvelsnaptracker#70]
Since the February 5, 2024 Android 14 security update, it's not possible to select the "nvprod" folder (or any folder).

To reproduce, open the tracker and click on "Locate Game Data".

![image](https://github.com/Razviar/marvelsnaptracker/assets/51435330/21b57de2-1f53-45f0-8120-d8101b930d17)

I'm running Android 14 on Google Pixel 7 Pro (Build number UQ1A.240205.002).

**Comment by Razviar** [comment]
yeah, basically Google killed trackers with this update. There's no working way to fix it. If anyone knows it, please let me know

**Comment by rene-j** [comment]
If you add "file access" permission to the app, would that solve it?

**Comment by Razviar** [comment]
No, we already asking for file access permission. But that's special system folder, which is strictly protected. There's an open issue on Adroid bug track: https://issuetracker.google.com/issues/256669329

**Comment by DanielRunningen** [comment]
Actually, if an APK is available for download, that would be great. All the play store links are busted and I'm still on a device that won't have this issue. I currently just see the Windows installer in the release artifacts.
